var/spack/repos/builtin/packages/ffmpeg/package.py

```diff
@@ -211,6 +212,9 @@ def configure_args(self):
         if spec.satisfies("@2.3:"):
             drawtext_opts.append("libfribidi")
 
+        if spec.satisfies("@7:"):
```

Comment:
Shouldn't `@7:` also be on the condition of the `depends_on` ?

Reply:
Fixed.